cmd/system-probe/modules/crashdetect_windows.go

```diff
@@ -0,0 +1,61 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build windows
+
+package modules
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/DataDog/datadog-agent/cmd/system-probe/api/module"
+	"github.com/DataDog/datadog-agent/cmd/system-probe/config"
+	"github.com/DataDog/datadog-agent/cmd/system-probe/utils"
+	"github.com/DataDog/datadog-agent/pkg/collector/corechecks/system/wincrashdetect/probe"
+	"github.com/DataDog/datadog-agent/pkg/ebpf"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// WinCrashProbe Factory
+var WinCrashProbe = module.Factory{
+	Name:             config.WindowsCrashDetectModule,
+	ConfigNamespaces: []string{"windows_crash_detection"},
+	Fn: func(cfg *config.Config) (module.Module, error) {
+		log.Infof("Starting the WinCrashProbe probe")
+		bsp, err := probe.NewWinCrashProbe(ebpf.NewConfig())
+		if err != nil {
+			return nil, fmt.Errorf("unable to start the BSOD probe: %w", err)
+		}
+		return &winCrashDetectModule{
+			WinCrashProbe: bsp,
```

Comment:
it was `blue screen probe`.  Changed to `cp`, for crash probe, which follows the convention that it's variable of type WinCrashProbe. I supose it could be `wcp`